ts/features/lollipop/saga/index.ts

```diff
@@ -110,10 +115,19 @@ export function* checkLollipopSessionAssertionAndInvalidateIfNeeded(
   );
 
   if (!lollipopCheckResult) {
+    void mixpanelTrack(
+      "LOGIN_UNEXPECTED_REQUEST_ID",
+      buildEventProperties("KO", undefined)
+    );
     yield* put(sessionInvalid());
+    yield* delay(WAIT_A_BIT_AFTER_SESSION_EXPIRED);
```

Comment:
Could we add a comment about this "delay", to remind us of its function in the future?